### PR TITLE
Add cs language

### PR DESCRIPTION
**Pull request by zen0bit**
Since I cannot create new language on weblate...

### CODE REVIEW

**Comment by zen0bit** [comment]
I did it by hand, not using weblate or poedit etc..

File: po/cs.po [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+msgid "Always on Top"
+msgstr "Vždy nahoře"
+
+#: src/menu/menu.c:730
+msgid ""
+msgstr "Posunout doleva"
+
+#: src/menu/menu.c:737
+msgid ""
+msgstr "Posunout doprava"
```
Comment (Consolatis) [inline po/cs.po:57]: `msgid` seems to be missing for these two.

Comment (zen0bit) [reply po/cs.po]: Overlooked

**Comment by 01micko** [comment]
LGTM

**Comment by Consolatis** [comment]
Thanks :)

**Comment by stefonarch** [comment]
> Since I cannot create new language on weblate...

Users have to ask for for upgrading permissions, as there were once issues, there is a info but most users don't see it. Upgraded now for you.

**Pull request merged**